PROJECT_PROGRESSION.md

```diff
@@ -213,7 +213,8 @@ Projects in this stage are not in active development. Their maintainers may infr
 
 #### Acceptance Criteria
 
-Projects may be granted Emeritus status via a 2/3 vote from the CPC and with approval from project ownership. In cases where there is a lack of project ownership, only a 2/3 vote from the CPC is required.
+Projects may be granted Emeritus status with the consent of the CPC and with approval from project ownership.
+In cases where there is a lack of project ownership, a 2/3 vote from the CPC is required.
```

Comment:
Clarifying the concept of "ownership" which isn't really defined anywhere:

```suggestion
In cases where the project maintainers aren't responding to the CPC for six (6) weeks after having been contacted through the appropriate channels about the CPC's intent to move the project to Emeritus stage, the CPC may proceed with the stage change without approval from the maintainers.
```

Reply:
I think I agree with this in general, but it feels like a slightly bigger change to add this level of specificity. Also, is it intentional that you're proposing dropping the 2/3 vote requirement for the no-maintainers/owners case?

Reply:
> I think I agree with this in general, but it feels like a slightly bigger change to add this level of specificity.

Would you prefer:

```suggestion
In cases where the project maintainers aren't responding after having been repeatedly contacted through appropriate channels about the CPC's intent to move the project to Emeritus stage, the CPC may proceed with the stage change without approval from the maintainers.
```

> Also, is it intentional that you're proposing dropping the 2/3 vote requirement for the no-maintainers/owners case?

Yes. I don't see the rationale for making a distinction between those two cases. If anything, Emeritus should be automatic for projects who are no longer maintained.